openslides_backend/services/auth/adapter.py

```diff
@@ -32,7 +38,9 @@ def authenticate(
             f"Start request to authentication service with the following data: {headers}"
         )
         try:
-            return self.auth_handler.authenticate(headers, cookies)
+            access_token = headers.get(HEADER_NAME, None)
+            cookie = cookies.get(COOKIE_NAME, "")
```

Comment:
Is the type correct (str not bytes)?

Reply:
I think so. The [requests-docs](https://requests.readthedocs.io/en/master/user/quickstart/#cookies) suggests so, I think.

```
Cookies
If a response contains some Cookies, you can quickly access them:

>>> url = 'http://example.com/some/cookie/setting/url'
>>> r = requests.get(url)

>>> r.cookies['example_cookie_name']
'example_cookie_value'
To send your own cookies to the server, you can use the cookies parameter:

>>> url = 'https://httpbin.org/cookies'
>>> cookies = dict(cookies_are='working')

>>> r = requests.get(url, cookies=cookies)
>>> r.text
'{"cookies": {"cookies_are": "working"}}'
```